EIPS/eip-1.md

```diff
@@ -185,6 +187,10 @@ The `created` header records the date that the EIP was assigned a number. Both h
 
 EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on.
 
+#### `adoptable` header
+
+EIPs may have a `adoptable` header. If this is `true`, then any PR changing `status` from `Stagnant` to a non-final state no longer requires author approval (instead only requiring editor approval.) Marking a proposal as `adoptable` allows _anyone_ to edit the author list and take over the proposal should it be abandoned, and is recommended.
```

Comment:
I'd love to see a more clear description on what is considered "abandoned"

Reply:
Stagnant <=> abandoned